src/components/application_manager/include/application_manager/rpc_handler_impl.h

```diff
@@ -145,6 +145,13 @@ class RPCHandlerImpl : public RPCHandler,
 #endif  // TELEMETRY_MONITOR
 
  private:
+  /**
+   * @brief Checks if message has to be sent to mobile or not
+   * @param output - message to check
+   * @returns true if message type is response otherwise false
+   */
+  bool HandleWrongMessageType(smart_objects::SmartObject& output) const;
```

Comment:
@ZhdanovP I believe this method could be const

Reply:
@AKalinich-Luxoft This method already const

Reply:
@ZhdanovP please add description for `report`